starter-code/pizza.js

```diff
@@ -1 +1,186 @@
 // Write your Pizza Builder JavaScript in this file.
```

Comment:
A way to make this code more efficient is to create a single function that by passing different buttons and classes as parameters handle the active state of an ingredient and it renders it or not depending on the status.
The price on the other way should be calculated in a separated way based on the active classes through a function, this function then will be called everytime an ingredient button will be clicked